L1Trigger/L1TMuonCPPF/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<export>
+  <lib name="1"/>
+</export>
+<flags ADD_SUBDIR="1"/>
```

Comment:
you do not need this line. It is needed only if you have source code in package/src/some-subdir. I do not see any subdir under your src, so no need to have this line.

Reply:
No, it's no necessary, I changed this line